plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -161,6 +161,11 @@ public IStatus validate(Object value /* email */) {
         .setConverter(new Converter(String.class, String.class) {
       @Override
       public Object convert(Object fromObject /* email */) {
+        // If there exists only one account signed in, make it selected.
+        String singleAccountEmail = accountSelector.getSingleAccountEmail();
```

Comment:
This converter doesn't seem the right location.  I expected to find logic in the `AccountSelector`.